apps/scale-count.html.markerb

```diff
@@ -5,15 +5,23 @@ layout: docs
 nav: firecracker
 order: 60
 ---
-<%= partial "/docs/partials/v2_transition_banner" %>
 
-<div class="callout">This document describes horizontal scaling of [V2 apps](/docs/reference/apps/#apps-v2), running on Fly Machines. 
+For Nomad apps see [scaling legacy V1 apps](/docs/apps/legacy-scaling/).
 
-**For existing Nomad/V1 Fly Apps:** There's a whole doc on [scaling legacy V1 apps](/docs/apps/legacy-scaling/).
+An app's horizontal scale, or the number of Fly Machines belonging to it, sets its maximum capacity. This is done by creating or destroying Machines.
```

Comment:
```suggestion
The maximum capacity of an app is determined by the number of Fly Machines belonging to it.  You scale an app horizontally by creating or destroying Machines.
```
I hope I'm keeping your meaning here. I'm not sure about "horizontal scale" as a noun? But will leave it up to you.